modules/delegation-migration/components/DelegateExpiryDate.tsx

```diff
@@ -0,0 +1,75 @@
+import { Text, Flex } from 'theme-ui';
+import React from 'react';
+import Icon from 'modules/app/components/Icon';
+import { DateWithHover } from 'modules/app/components/DateWithHover';
+import { Delegate } from 'modules/delegates/types';
+
+export default function DelegateExpiryDate({ delegate }: { delegate: Delegate }): React.ReactElement {
+  const iconStyles = {
+    orangeCalendar: {
+      fill: '#F75524',
+      stroke: '#F75524'
+    },
+    redCalendar: {
+      fill: '#FF0000',
+      stroke: '#FF0000'
+    },
+    activeCalendar: {
+      fill: 'primary',
+      stroke: 'primary'
+    }
+  };
```

Comment:
Do we not have variables for these colors? We should use whatever is closest instead of always adding slight variations.